rust/operator-binary/src/airflow_controller.rs

```diff
@@ -1170,7 +1170,7 @@ fn build_server_rolegroup_statefulset(
             match_labels: Some(statefulset_match_labels.into()),
             ..LabelSelector::default()
         },
-        service_name: Some(rolegroup_ref.object_name()),
+        service_name: Some(format!("{name}-metrics", name = rolegroup_ref.object_name())),
```

Comment:
Just curious, is there a more authoritative source for the service name (eg: a function, in case it is needed in multiple places)?